### PR TITLE
Add ForgotPassword component

### PR DESCRIPTION
**Pull request by VitaliKozhurov**

### CODE REVIEW

File: src/features/auth/ui/forgotPasswordForm/ForgotPasswordForm.tsx [hunk 1]
```diff
@@ -8,8 +10,8 @@ type Props = {
   onSubmit: (data: ForgotPasswordFormValue) => void
 }
 
-export const ForgotPassword = ({ onSubmit }: Props) => {
-  const { control, handleSubmit } = useForgotPassword()
+export const ForgotPasswordForm = ({ onSubmit }: Props) => {
```
Comment (vitalyabedik) [inline src/features/auth/ui/forgotPasswordForm/ForgotPasswordForm.tsx:13]: JSX.Element

Comment (VitaliKozhurov) [reply src/features/auth/ui/forgotPasswordForm/ForgotPasswordForm.tsx]: Done!

**Review by vitalyabedik — APPROVED** [review]
Good!

File: src/pages/forgotPassword/ForgotPassword.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { JSX } from 'react'
```
Comment (DenisKulik) [inline src/pages/forgotPassword/ForgotPassword.tsx:1]: Why this import?

**Review by DenisKulik — APPROVED** [review]

File: src/pages/forgotPassword/ForgotPassword.tsx (hunk 2 above)
Comment (VitaliKozhurov) [reply src/pages/forgotPassword/ForgotPassword.tsx]: Remove import!

**Pull request merged**